### PR TITLE
chore: update adopters, use cases and industry page

### PR DESCRIPTION
**Pull request by hacktivist123**
This PR updates the following pages with new content:

- adopters page with new logos from Roche and Sicredi. 
- added additional links for new york times and Microsoft logos in the adopters page
- Bumped up Schuberg Philis to the top section of the adopters page
- cluster mesh use case page
- service mesh use case page
- financial services industry page

### CODE REVIEW

File: src/utils/case-studies-data.js [hunk 1]
```diff
@@ -845,6 +855,39 @@ export default {
...
+      {
+        linkText: 'Watch the Talk',
+        linkUrl: 'https://youtu.be/MSqI-gBiCrc?si=I8QMmlSB7cnYe0BQ',
+        linkTarget: '_blank',
+      },
+    ],
+  },
+  schubergPhilis: {
+    iconName: 'schubergPhilis',
+    text: '<b>Schuberg Philis</b> uses Cilium as the CNI for mission critical Kubernetes clusters they run for their customers',
```
Comment (xmulligan) [inline src/utils/case-studies-data.js:871]:
```suggestion
    text: '<b>Schuberg Philis</b> uses Cilium as the CNI for their PCI-DSS Certified Kubernetes clusters',
```

File: src/pages/industries/financial-services.jsx [hunk 2]
```diff
@@ -180,6 +181,15 @@ const financialResources = [
...
       "Robinhood's war stories from running Cilium in a high-churn near-production environment, learn how they overcame challenges by better understanding and tuning Cilium.",
   },
+  {
+    imageSrc: SicrediOfficeImage,
+    imageAlt: 'Sicredi logo on a wall',
+    title: 'Strengthening Security Across Distributed Kubernetes Clusters',
+    buttonLink: 'https://www.youtube.com/watch?v=MSqI-gBiCrc',
+    buttonText: 'Watch the Talk',
+    description:
+      "Sicredi, Brazil's largest credit union, leveraged Cilium to implement a full security policy across its infrastructure.",
```
Comment (xmulligan) [inline src/pages/industries/financial-services.jsx:191]: What does this mean?

File: src/pages/use-cases/cluster-mesh.jsx [hunk 3]
```diff
@@ -131,6 +132,24 @@ const testimonials = [
...
+        CTAtext: 'Watch The Talk',
+        url: 'https://youtu.be/MSqI-gBiCrc?si=ktIHXkxhuHpuwC6a',
+      },
+      // {
+      //   CTAtext: 'Read Case Study',
+      //   url: '',
+      // },
+    ],
+    description:
+      "Cilium Cluster Mesh gave us possibilities that we didn't have before. We can run the same application across our data centers and AWS. It provides us with a consistent networking experience wherever we need to go. Applications in different clusters can communicate with each other without needing to go through an ingress controller. We are expanding this capability for more applications because this value is already proven. Cluster Mesh works, it's fast, and it's reliable",
```
Comment (xmulligan) [inline src/pages/use-cases/cluster-mesh.jsx:149]: really good quote choice

File: src/pages/use-cases/service-mesh.jsx [hunk 4]
```diff
@@ -51,13 +54,47 @@ const sectionContent4 = {
...
+    title: 'Strengthening Security Across Distributed Kubernetes Clusters',
+    CTAs: [
+      {
+        CTAtext: 'Watch The Talk',
+        url: 'https://youtu.be/MSqI-gBiCrc?si=ktIHXkxhuHpuwC6a',
+      },
+      // {
+      //   CTAtext: 'Read Case Study',
+      //   url: '',
+      // },
```
Comment (xmulligan) [inline src/pages/use-cases/service-mesh.jsx:81]: should this be in here?

File: src/pages/use-cases/service-mesh.jsx [hunk 5]
```diff
@@ -51,13 +54,47 @@ const sectionContent4 = {
...
+    ],
+    description:
+      "We started using Cilium in our EKS clusters and encountered security challenges. So many microservices were running without any kind of authentication or encryption. This prompted us to start doing a service mesh proof of concept. Although we already had Cilium available, we also evaluated Consul Connect, Kong Mesh, and Istio. Cilium stood out as it did not require sidecars, appealing to us not only for resource savings but also to avoid potential load issues caused by additional software. Cilium's sidecar-less approach did call our attention, leading us to adopt it to provide security across all our microservices.",
+    quote: true,
+    person: 'Matheus Morais - IT Infrastructure Analyst, Sicredi',
+  },
+];
+
```
Comment (xmulligan) [inline src/pages/use-cases/service-mesh.jsx:89]: you should also add the NYT talk to the service mesh page

Comment (hacktivist123) [reply src/pages/use-cases/service-mesh.jsx:89]: Alright, would do!

File: src/pages/use-cases/cluster-mesh.jsx (hunk 3 above)
Comment (hacktivist123) [reply src/pages/use-cases/cluster-mesh.jsx:149]: I know right, it felt perfect for that page

File: src/pages/industries/financial-services.jsx (hunk 2 above)
Comment (hacktivist123) [reply src/pages/industries/financial-services.jsx:191]: Sicredi used Cilium cluster mesh mainly for service mesh, cluster mesh and security that came with it.  I wanted to compress all that detail into a one liner and also got that from their talk description at kubecon

File: src/utils/case-studies-data.js (hunk 1 above)
Comment (hacktivist123) [reply src/utils/case-studies-data.js]: updated

**Comment by hacktivist123** [comment]
added eBay to network policy page and NY Times to service mesh page

File: src/pages/use-cases/service-mesh.jsx [hunk 6]
```diff
@@ -51,13 +55,59 @@ const sectionContent4 = {
...
+    logo: RocheLogo,
+    title: 'How Roche Manages Network Connectivity for 1000+ Edge Clusters',
+    CTAs: [
+      {
+        CTAtext: 'Watch The Talk',
+        url: 'https://www.youtube.com/watch?v=vKgRf4OzTIE',
+      },
+    ],
+    description:
+      "The Roche team utilized Cilium's service mesh as a solution for managing network connectivity across their edge clusters. Cilium enabled them to have more fine-grained traffic control, and simplified the operations and configuration of network policies using a GitOps approach, alongside the deployment of workloads",
```
Comment (xmulligan) [inline src/pages/use-cases/service-mesh.jsx:69]:
```suggestion
      "The Roche team utilized Cilium's service mesh as a solution for managing network connectivity across their edge clusters. Cilium enabled them to have more fine-grained traffic control, and simplified the operations and configuration of network policies using a GitOps approach, alongside the deployment of workloads.",
```

File: src/pages/industries/financial-services.jsx (hunk 2 above)
Comment (xmulligan) [inline src/pages/industries/financial-services.jsx:191]:
```suggestion
      "Sicredi, Brazil's largest credit union, leverages Cilium to reduced operational and maintenance complexity while increasing performance across clouds and on premise.",
```

**Review by xmulligan — APPROVED** [review]

**Pull request merged**